src/main/kotlin/no/nav/familie/ba/mottak/hendelser/LeesahService.kt

```diff
@@ -121,7 +123,21 @@ class LeesahService(
                     fødselIgnorertCounter.increment()
                 }
             }
-
+            ANNULLERT -> {
+                fødselAnnullertCounter.increment()
+                if (pdlHendelse.tidligereHendelseId != null) {
+                    val task = Task.nyTask(type = MottaAnnullerFødselTask.TASK_STEP_TYPE,
+                                           payload = objectMapper.writeValueAsString(pdlHendelse.hentPersonidenter()),
+                                           properties = Properties().apply {
+                                               this["identer"] = objectMapper.writeValueAsString(pdlHendelse.hentPersonidenter())
```

Comment:
I think toString() is enough or just choose the first of them. This info is only used in familie-prosessering to analyse task

Reply:
Ok